src/components/GlobalPreferences/Notifications/NotificationsLogin.js

```diff
@@ -0,0 +1,179 @@
+  )
+  const handleEmailBlur = useCallback(
+    e => {
+      const email = e.target.value
+      setEmailInvalid(!validateEmail(email))
+    },
+    [setEmailInvalid]
+  )
+
+  const handleEmailChange = useCallback(
+    e => {
+      const email = e.target.value
+      setInputEmail(email)
+      handleEmailValidation(email)
+    },
+    [handleEmailValidation, setInputEmail]
+  )
+
+  const handleLogin = async e => {
+    e && e.preventDefault()
+    try {
+      await login({ email: inputEmail, dao, network: ethNetwork })
+      onEmailChange(inputEmail)
+    } catch (e) {
+      setApiError(e.message)
+      console.error('Failed to login', e)
+    }
+  }
+
+  const theme = useTheme()
+
+  return (
+    <Box heading="Email notifications">
+      <NotificationImage />
+      <form onSubmit={handleLogin}>
+        <div
+          css={`
+            display: grid;
+            align-items: end;
+            grid-template-columns: 1fr auto;
+            grid-gap: ${GU}px ${2 * GU}px;
+            margin-bottom: ${2 * GU}px;
+          `}
+        >
+          <label
+            css={`
+              color: ${theme.surfaceContentSecondary};
+              display: block;
+            `}
+          >
+            Email address
+            <TextInput
+              css={`
+                border-color: ${emailInvalid === true
+                  ? theme.error
+                  : theme.border};
+              `}
+              adornment={
+                emailInvalid === false ? (
+                  <IconCheck
+                    css={`
+                      color: ${theme.positive};
+                    `}
+                  />
+                ) : inputEmail.trim() ? (
+                  <IconCross
+                    css={`
+                      color: ${theme.negative};
+                    `}
+                  />
+                ) : null
+              }
+              adornmentPosition="end"
+              type="email"
+              placeholder="you@example.com"
+              wide
+              value={inputEmail}
+              onChange={handleEmailChange}
+              onBlur={handleEmailBlur}
+            />
+          </label>
+
+          <Button
+            disabled={emailInvalid !== false}
+            css={`
+              width: ${16 * GU}px;
+            `}
+            onClick={handleLogin}
+            icon={<IconMail />}
+            label="Sign in"
+          />
+          {apiError && (
+            <p
+              css={`
+                color: ${theme.negative};
+                ${textStyle('body2')};
+              `}
+            >
+              Error logging in
+            </p>
+          )}
+          {emailInvalid && (
+            <p
+              css={`
+                color: ${theme.negative};
+                ${textStyle('body2')};
+              `}
+            >
+              Please enter a valid email address
+            </p>
+          )}
+        </div>
+      </form>
+
+      <Info>
```

Comment:
@dizzypaty would it make sense to split this in 2 info boxes? Feels a bit crowded it does communicate two different messages: what is this and how this works on diff devices with no password